com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/font/Fontc.java

```diff
@@ -118,6 +121,16 @@ public CompositeContext createContext(ColorModel arg0, ColorModel arg1, Renderin
 
 public class Fontc {
 
+    public static final char[] ASCII_7BIT;
+    static {
+        int start = 32;
+        int end = 126;
+        ASCII_7BIT = new char[end - start + 1];
+        for (int i = start; i <= end; i++) {
+            ASCII_7BIT[i - start] = (char) i;
+        }
+    }
```

Comment:
It would be better to have only one place where default symbols specified

Reply:
I know that we usually don't do static initialization like that, but in this case, having it like this explains the logic behind this string. If you don't mind, I keep it this way.